app/main/main.js

```diff
@@ -99,12 +104,19 @@ function registerFileProtocol(scheme, mapper) {
   });
 }
 
+const appStartupTime = new Date();
```

Comment:
This needs to go right up to the top before any require statements to give us a full measure